frontend/public/components/overview/namespace-overview.tsx

```diff
@@ -96,13 +129,29 @@ const OverviewResourceQuotas = ({ns}) => {
   </Firehose>;
 };
 
-export const OverviewNamespaceDashboard = ({obj: ns}) => <div className="co-m-pane__body">
+const OverviewNamespaceDashboard_ = ({obj: ns, consoleLinks}) => <div className="co-m-pane__body">
   <OverviewHealth ns={ns} />
   <OverviewResourceQuotas ns={ns} />
   <OverviewResourceUsage ns={ns} />
+  <OverviewLinks ns={ns} consoleLinks={consoleLinks} />
```

Comment:
It would be better to connect `OverviewLinks` instead of the entire `OverviewNamespaceDashboard`. You generally want to connect only the child component that needs the data when possible.